packages/contracts-bedrock/package.json

```diff
@@ -16,19 +16,20 @@
   "scripts": {
     "build:forge": "forge build",
     "build:differential": "tsc scripts/differential-testing.ts --outDir dist --moduleResolution node --esModuleInterop",
+    "build:scripts": "(cd scripts/merkle-trie-fuzzer && go build main.go)",
     "prebuild": "yarn ts-node scripts/verify-foundry-install.ts",
-    "build": "hardhat compile && yarn autogen:artifacts && yarn build:ts && yarn typechain",
+    "build": "hardhat compile && yarn autogen:artifacts && yarn build:ts && yarn typechain && yarn build:scripts",
     "build:ts": "tsc -p tsconfig.json",
     "autogen:artifacts": "ts-node scripts/generate-artifacts.ts",
     "deploy": "hardhat deploy",
-    "test": "yarn build:differential && forge test",
-    "coverage": "yarn build:differential && forge coverage",
-    "coverage:lcov": "yarn build:differential && forge coverage --report lcov",
-    "gas-snapshot": "yarn build:differential && forge snapshot",
+    "test": "yarn build:differential && yarn build:scripts && forge test",
```

Comment:
I haven't tested, but it feels like this might slow down the test run if you have to do this every time? If so, perhaps we could just throw an error if the script isn't built yet.

Reply:
At the moment, compilation of the binary takes less than a second, so it doesn't introduce much of a slowdown. Once this package is expanded, it might be a better idea to go this route.